docs/workflow/building/libraries/README.md

```diff
@@ -176,6 +176,17 @@ dotnet build System.Net.NetworkInformation.csproj /p:TargetOS=Linux
 dotnet build -c Release System.Net.NetworkInformation.csproj
 ```
 
+### Iterating on System.Private.CoreLib changes
+When changing `System.Private.CoreLib` after a full build, in order to test against those changes, you will need an updated `System.Private.CoreLib` in the testhost. In order to achieve that, you can build the `libs.pretest` subset which does testhost setup including copying over `System.Private.CoreLib`.
+
+```
+build.cmd clr.corelib+clr.nativecorelib+libs.pretest -rc Release
```

Comment:
I still have to build the runtime first, right? Maybe this should make clear that you would likely do a command like XYZ first, then repeat the command here as you iterate on S.P.C.

Reply:
Yeah, you should have done `build.cmd clr` or `build.cmd clr.runtime` first. I'll add a comment to clarify that.